newrelic/resource_newrelic_nrql_alert_condition.go

```diff
@@ -412,9 +413,25 @@ func resourceNewRelicNrqlAlertConditionCreate(ctx context.Context, d *schema.Res
 		return diag.FromErr(err)
 	}
 
-	d.SetId(serializeIDs([]int{d.Get("policy_id").(int), conditionID}))
+	retryErr := resource.RetryContext(ctx, d.Timeout(schema.TimeoutCreate), func() *resource.RetryError {
```

Comment:
Should we define a value for `schema.TimeoutCreate`? I wasn't sure if there was a default value.

Reply:
I think it has a default, but I added it to be sure. Thanks 👍